### ISSUE TITLE
error raised by passing axis=None to tick_params()

### ISSUE THREAD

**Issue by t-silvers** [AlexandrovLab/SigProfilerPlotting#6]
Hi, In `sample_portrait.py`, it looks like `Axes.tick_params(axis=None)` is used a few times, which is invalid and raises an error in `matplotlib>=3.1.0` ([see here](https://matplotlib.org/3.1.1/api/api_changes.html#axes-tick-params-argument-checking)). It would be great to address the underlying issue, but in the meantime you could add `install_requires` with working versions in your setup.

See below for error, in case I'm missing something.
```
 sP.samplePortrait(project_dir, project_dir, 'project', percentage=False)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-20-ced65c03fc90> in <module>
----> 1 sP.samplePortrait(project_dir, project_dir, 'project', percentage=False)

./conda/4.6.14/envs/sigtest/lib/python3.8/site-packages/sigProfilerPlotting/sample_portrait.py in samplePortrait(sample_matrices_path, output_path, project, percentage)
   1474
   1475
-> 1476                 panel5.tick_params(axis=None,which='both',\
   1477                                                    bottom=False, labelbottom=False,\
   1478                                                    left=False, labelleft=False,\

./conda/4.6.14/envs/sigtest/lib/python3.8/site-packages/matplotlib/axes/_base.py in tick_params(self, axis, **kwargs)
   2944
   2945         """
-> 2946         cbook._check_in_list(['x', 'y', 'both'], axis=axis)
   2947         if axis in ['x', 'both']:
   2948             xkw = dict(kwargs)

./conda/4.6.14/envs/sigtest/lib/python3.8/site-packages/matplotlib/cbook/__init__.py in _check_in_list(_values, **kwargs)
   2141     for k, v in kwargs.items():
   2142         if v not in values:
-> 2143             raise ValueError(
   2144                 "{!r} is not a valid value for {}; supported values are {}"
   2145                 .format(v, k, ', '.join(map(repr, values))))

ValueError: None is not a valid value for axis; supported values are 'x', 'y', 'both'

```

**Comment by t-silvers** [comment]
Here's an env file that ended up working for me

```
name: sigprofiler
channels:
  - conda-forge
  - bioconda
  - defaults
dependencies:
  - python=3.6
  - matplotlib=2.2.3
  - pip
  - pip:
    - SigProfilerPlotting
    - SigProfilerMatrixGenerator
```

**Comment by ebergstr** [comment]
Thank you for pointing this issue out! It has been fixed in version 1.1.6
